Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -284,44 +290,78 @@ def unit_tests_results() -> list[dict[str, Any]]:
 
 def bucket_upload_results(bucket_artifact_folder: Path,
                           marketplace_name: str,
-                          should_include_private_packs: bool) -> list[dict[str, Any]]:
-    steps_fields = []
+                          should_include_private_packs: bool) -> tuple[list[dict[str, Any]], list[dict[str, Any]]]:
+    slack_msg_append = []
+    threaded_messages = []
     pack_results_path = bucket_artifact_folder / BucketUploadFlow.PACKS_RESULTS_FILE_FOR_SLACK
 
     logging.info(f'retrieving upload data from "{pack_results_path}"')
     successful_packs, _, failed_packs, successful_private_packs, _ = get_upload_data(
         pack_results_path.as_posix(), BucketUploadFlow.UPLOAD_PACKS_TO_MARKETPLACE_STORAGE
     )
     if successful_packs:
-        steps_fields += [{
-            'title': f'Successful {marketplace_name} Packs:',
-            'value': ', '.join(sorted({*successful_packs}, key=lambda s: s.lower())),
-            'short': False
-        }]
-
-    if failed_packs:
-        steps_fields += [{
-            'title': f'Failed {marketplace_name} Packs:',
-            'value': ', '.join(sorted({*failed_packs}, key=lambda s: s.lower())),
-            'short': False
-        }]
+        slack_msg_append.append({
+            'fallback': f'Successfully uploaded {len(successful_packs)} Pack(s) to {marketplace_name}',
+            'title': f'Successfully uploaded {len(successful_packs)} Pack(s) to {marketplace_name}',
+            'color': 'good',
+        })
+        threaded_messages.append({
+            'fallback': f'Successfully uploaded {marketplace_name} Pack(s): '
+                        f'{", ".join(sorted({*successful_packs},key=lambda s: s.lower()))} to {marketplace_name}',
+            'title': f'Successfully uploaded {len(successful_packs)} Pack(s) to {marketplace_name}.',
+            'color': 'good',
+            'fields': [{
+                'title': 'Pack(s):',
```

Comment:
Not sure if this title is relevant, WDYT?

Reply:
i removed the title. but i will keep the title empty, as it's mandatory field.